src/cmd/prune_branches.go

```diff
@@ -31,10 +42,14 @@ This usually means the branch was shipped or killed on another machine.`,
 	},
 }
 
-func checkPruneBranchesPreconditions() {
+func checkPruneBranchesPreconditions() error {
 	if git.HasRemote("origin") {
-		script.Fetch()
+		err := script.Fetch()
+		if err != nil {
+			return err
+		}
```

Comment:
I think this could be simplified to 
```go
return script.Fetch()
```
(removing the need to check whether on not err is nit

Reply:
That's correct. Good catch!